worker-versioning/README.md

```diff
@@ -0,0 +1,18 @@
+# Build ID Based Versioning
+
+This sample illustrates how to use Build ID based versioning to help you appropriately roll out
```

Comment:
```suggestion
This sample illustrates how to use [Build ID based versioning](https://docs.temporal.io/workers#worker-versioning) to help you appropriately roll out
```

Reply:
might be a little confusing that we have 2 names for this? thought it was called "Worker Versioning"

Reply:
Yeah, we do, this is a copy-paste error I think